htdocs/mri/jiv/get_file.php

```diff
@@ -57,9 +57,11 @@
     exit(2);
 }
 
-// Now get the file and do file validation
-$File = $_GET['file'];
+// Now get the file and do file validation.
+// Resolve the filename before doing anything.
+$File = realpath($_GET['file']);
```

Comment:
No it's not working for me either... I guess I only checked to see whether the exploit worked, not the actual downloading... @driusan we'll probably have to roll this back